nuxt.config.js

```diff
@@ -187,12 +204,41 @@ module.exports = {
           enforce: 'pre',
           test: /\.(js|vue)$/,
           loader: 'eslint-loader',
-          exclude: /(node_modules)/
+          exclude: /(node_modules)/,
+          options: {
+            fix: true
+          }
```

Comment:
What does this do? An auto fix on save?

Reply:
Yes

Reply:
@annachristaki could you remove it, and just run npm lint before you commit?

Reply:
Agreed. Please do so.

Reply:
Agreed. Please do so.